platform/platform-configuration-migration/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigrationTest.java

```diff
@@ -498,32 +462,26 @@ public void testInitMovesFilesWithAnInvalidTypeToFailedDirectory() throws Except
         when(configurationFileFactory.createConfigurationFile(configPath2)).thenThrow(new ConfigurationFileException(
                 ""));
 
-        ConfigurationAdminMigrationUnderTest configurationAdminMigrator =
-                new ConfigurationAdminMigrationUnderTest(configurationDirectoryStream,
-                        processedDirectoryPath,
-                        failedDirectoryPath,
-                        configurationFileFactory,
-                        configurationFilePoller,
-                        configurationAdmin,
-                        configurationFileExtension);
+        ConfigurationAdminMigration configurationAdminMigration = new ConfigurationAdminMigration(
+                configurationDirectoryStream,
+                processedDirectoryPath,
+                failedDirectoryPath,
+                configurationFileFactory,
+                configurationFilePoller,
+                configurationAdmin,
+                configurationFileExtension);
 
-        configurationAdminMigrator.init();
+        configurationAdminMigration.init();
 
-        assertThat("Too many files moved",
-                configurationAdminMigrator.filesMoved.keySet(),
-                hasSize(2));
-        assertThat("Configuration file 1 not moved to the failed directory",
-                configurationAdminMigrator.filesMoved,
-                hasEntry(configPath1, failedDirectoryPath));
-        assertThat("Configuration file 2 not moved to the failed directory",
-                configurationAdminMigrator.filesMoved,
-                hasEntry(configPath2, failedDirectoryPath));
+        verify(configurationFileFactory).createConfigurationFile(configPath1);
+        verify(configurationFileFactory).createConfigurationFile(configPath2);
```

Comment:
We need to verify that the files have been moved to the failed directory here, like we did in `testInitMovesFilesToProcessedDirectory()`.